ldb-eda-structure.py

```diff
@@ -0,0 +1,505 @@
+#!/usr/bin/env python
```

Comment:
If this pull request it's about data cleanning this file shouldn't be here.

Reply:
Done!